code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -106,7 +106,7 @@
 	name = "loyalty"
 	desc = "Courtesy of the 16th Ego rifleman's brigade."
 	icon_state = "loyalty"
-	inhand_icon_state = "beak"
+	inhand_icon_state = "loyalty"
```

Comment:
That. That Explains a Lot.

Reply:
Yeah Originally it didn't have an in-hand, and I did it way after I made the gun and popped the files in.
Forgot to change the inhand state though.